nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -0,0 +1,312 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nvidia.dali as dali
+from nvidia.dali.pipeline import Pipeline
+from nvidia.dali.plugin.pytorch import DALIGenericIterator as DALIPytorchIterator
```

Comment:
The above two imports will fail unless Nemo auto installs Dali. As Dali setup is slightly different from standard pip install, please put a try catch here (checking for import error, module not found error) and raise an exception inside the constructor of the AudioToCharDaliDataeet class if the import failed.